### DISCUSSION TITLE
Releases are not displayed on the GitHub Repo page

### DISCUSSION THREAD

**Discussion by prajwalkpatil** [community/community#62048] [Repositories] [answered]
### Select Topic Area

Bug

### Body

When I check the box to display the releases on the front page of the repository, the releases are not displayed.
I have made one release. The release section is simply missing even after explicitly checking the box to display it.

## Before:
![Web capture_28-7-2023_16944_github com](https://github.com/community/community/assets/32927982/2d5cdf9a-9e9d-465f-a8ed-3cdddc819174)

## Editing:
![checking](https://github.com/community/community/assets/32927982/ff13332c-69ac-42b5-92c7-d38d6f2dfe14)

## After:
![Web capture_28-7-2023_161126_github com](https://github.com/community/community/assets/32927982/becbdc5f-90b4-40d1-9896-04f54148e57f)

_Note: This might have occured due to the fact that I changed the visibility of the repository before making that release._

Thank you.

**Comment by CodeMaster7000** [comment]
@prajwalkpatil Hello there and hope you are having a great day,

It is very clear that you have released 1.0 (https://github.com/prajwalkpatil/kanban/releases). The reason for this is most likely the change in visibility of the repository, but if the issue still persists you should contact GitHub support.

Thank you and good luck!

**Reply by prajwalkpatil** [reply]
Ok. Thank you.

**Reply by prajwalkpatil** [reply]
@CodeMaster7000  If you can tell me under which section I am supposed to report this or share a link for that, it will be a great help.

Thank you.

**Reply by CodeMaster7000** [reply, marked as answer]
If you do wish to report this to GitHub, you can create a support ticket by following the list of procedures [here](https://docs.github.com/en/support/contacting-github-support/creating-a-support-ticket).
